src/inverted_index.c

```diff
@@ -37,7 +40,7 @@ static t_docId calculateId(t_docId lastId, uint32_t delta, int isFirst);
 
 /* Add a new block to the index with a given document id as the initial id */
 IndexBlock *InvertedIndex_AddBlock(InvertedIndex *idx, t_docId firstId) {
-
+  TotalIIBlocks++;
```

Comment:
Never decremented. Is this intentional?

Reply:
Yes cause we never really remove block, we just clear their content. This might be able to let as know if this is the leak everyone experience lately. If the inverted_sz_mb is small but there is a lot of block it means that there is a lot of empty block.

Reply:
Decrement it in II_Free, it can't hurt, even if it's only called during drop/flush/etc.

Reply:
done